src/ros_can/src/node/node_ros_can.cpp

```diff
@@ -568,3 +540,12 @@ void RosCan::motor_speed_publisher(const unsigned char msg[8]) {
   // RCLCPP_DEBUG(this->get_logger(), "Received motor speed from Bamocar: %d", this->motor_speed_);
   motor_rpm_pub_->publish(message);
 }
+
+
+void RosCan::set_as_off_go_signal(){
```

Comment:
These functions are only used for tests. Therefore, they should not be here. Remove them and exchange them for direct accesses, as the tests are already friend tests